ipn/localapi/localapi.go

```diff
@@ -914,6 +915,23 @@ func (h *Handler) serveFileTargets(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(fts)
 }
 
+func (h *Handler) serveFlushLogtail(w http.ResponseWriter, r *http.Request) {
+	if !h.PermitWrite {
+		http.Error(w, "access denied", http.StatusForbidden)
+		return
+	}
+	if r.Method != "POST" {
+		http.Error(w, "want POST", http.StatusMethodNotAllowed)
+		return
+	}
+	if h.b.TryFlushLogs() {
+		w.Header().Set("Content-Type", "application/json")
```

Comment:
`w.WriteHeader(http.StatusNoContent)` seems to be the more common way in which LocalAPI handlers indicate success with no body.

Reply:
in this case I'm imagining including content here later, like some stats on what was buffered